individual-files/ControlFlow.java

```diff
@@ -0,0 +1,45 @@
+import java.io.IOException;
+import java.util.logging.Logger;
+
+public class ControlFlow
+{
+    public ControlFlow()
+    {
+        byte inputBuffer[] = new byte[ 128 ];
```

Comment:
Omission (O) - missing library require to use byte[] : import java.io.InputStream.*;